assets/src/editorIndex.js

```diff
@@ -47,3 +48,18 @@ replaceTaxonomyTermSelectors();
 setupCustomSidebar();
 setUpCssVariables();
 blockEditorValidation();
+
+const { registerBlockVariation } = wp.blocks;
+const { __ } = wp.i18n;
+
+registerBlockVariation('core/group', {
+  name: 'group-stretched-link',
+  title: __('Stretched Link', 'planet4-blocks-backend'),
+  description: __('Make the entire block contents clickable, using the first link inside.', 'planet4-blocks-backend'),
+  attributes: { className: 'group-stretched-link' },
+  scope: ['inserter', 'transform'],
+  isActive: (blockAttributes) => {
+    return blockAttributes.className === 'group-stretched-link';
+  },
+  icon: <Fragment>🔗</Fragment>,
```

Comment:
We could use the [admin-links](https://developer.wordpress.org/resource/dashicons/#admin-links) dash icon for visual consistency.

Reply:
Updated! 👍